client/tests/webdriver/baseSpecs/createNewPosition.spec.js

```diff
@@ -21,6 +21,20 @@ describe("Create position page", () => {
       CreatePosition.organizationHelpBlock.waitForExist()
       CreatePosition.organizationHelpBlock.waitForDisplayed()
 
+      CreatePosition.typeAdvisorButton.click()
+
+      expect(CreatePosition.positionNameHelpBlock.getText()).to.equal(
+        "Position name is required"
+      )
+      expect(CreatePosition.organizationHelpBlock.getText()).to.equal(
+        "Organization is required"
+      )
+      expect(CreatePosition.locationHelpBlock.getText()).to.equal(
+        "Location is required"
```

Comment:
Also update this to `"Location is required for an advisor position"` if you make the recommended change above.